Source/WebCore/platform/cocoa/PlaybackSessionModel.h

```diff
@@ -61,7 +62,9 @@ enum class PlaybackSessionModelPlaybackState : uint8_t {
     Stalled = 1 << 1,
 };
 
-class PlaybackSessionModel : public CanMakeWeakPtr<PlaybackSessionModel> {
+class PlaybackSessionModel : public CanMakeWeakPtr<PlaybackSessionModel>, public CanMakeCheckedPtr<PlaybackSessionModel> {
```

Comment:
It's already subclasses `CanMakeWeakPtr`, I would just use a WeakPtr / WeakRef instead of a CheckedPtr / CheckedRef.